packages/syft/tests/syft/request/request_multiple_nodes_test.py

```diff
@@ -149,6 +150,7 @@ def compute_sum(data) -> float:
 
 
 @pytest.mark.flaky(reruns=2, reruns_delay=1)
+@pytest.mark.skipif(sys.platform == "win32", reason="very flaky on windows")
```

Comment:
This should be in a separate PR. You can create a separate PR for this and post it in the engineering channel to have others' opinion.

Reply:
Thanks sure @kiendang